src/coreclr/jit/gentree.h

```diff
@@ -7369,6 +7345,67 @@ struct GenTreeVecCon : public GenTree
 #endif
 };
 
+// GenTreeMskCon -- mask constant (GT_CNS_MSK)
+//
+struct GenTreeMskCon : public GenTree
```

Comment:
should this be under `FEATURE_MASKED_HW_INTRINSICS` too?

Reply:
This is matching how `GenTreeVecCon` exists, which is likewise not under `FEATURE_SIMD`.

AFAIR this is because it makes checks around `GT_CNS_MSK` and `GT_CNS_VEC` much simpler since there is a guaranteed ID for them, they just won't be encountered. If we did put it under an ifdef, then all the places that handle `GT_CNS_VEC` including places like `OperIsConst` would need to start being ifdef'd as well.

It might be worth doing this long term, but I'd rather investigate such cleanup in a separate PR.